docs/install_guide.rst

```diff
@@ -0,0 +1,122 @@
+Installation guide - Docker compose
+===================================
+
+This document provides instructions on how to create a trivial deployment
+environment on single host for dojot, using docker-compose as the processes
+orchestration platform.
+
+While very simple, this deployment option is best suited to development and
+assessment of the platform and should not be used for production environments.
+
+This guide has been checked on an Ubuntu 16.04 LTS environment.
+
+.. contents:: Table of Contents
+  :local:
+
+Dependencies
+------------
+
+This setup has two software requirements docker engine and docker-compose.
+
+Docker engine
+^^^^^^^^^^^^^
+
+Up to date information and installation procedures for the docker engine can
+be found at the project's documentation:
+
+https://docs.docker.com/engine/installation/
+
+.. note::
+
+  An optional step on the installation and configuration process of docker on any given
+  machine is the setting of who is eligible for creating/spawning docker instances.
+
+  Should the post-installation steps (more specifically the "Manage docker as non-root user")
+  have not been run, all docker and docker-compose commands should be run by the super user (root),
+  or as sudo.
+
+  https://docs.docker.com/engine/installation/linux/linux-postinstall/
+
+Docker Compose
+^^^^^^^^^^^^^^
+
+Up to date information and installation procedures for the docker engine can
+be found at the project's documentation:
+
+https://docs.docker.com/compose/install/
+
+Installation
+------------
+
+To setup the environment, merely clone the deployment repository and run the
+commands below.
+
+The docker-compose enabled deployment scripts and configuration repository
+can be found at:
+
+https://github.com/dojot/docker-compose
+
+or as git clone command:::
+
+  git clone git@github.com:dojot/docker-compose.git
+
+Once the repository is properly cloned, select the version to be used by
+checking out the appropriate tag (do notice that the tagname has to be replaced): ::
+
+  # Must be run from within the deployment repo
+  git checkout [tag name]
+
+That done, the environment can be brought up by: ::
+
+  # Must be run from the root of the deployment repo.
+  # May need sudo to work: sudo docker-compose up -d
+  docker-compose up -d
+
+.. note::
+
+  Should the pulling of the container a container image fail, adding the explicit
+  reference to the internal registry may solve the problem.
+
+  Should that suffice or not, be sure to open a JIRA ticket for that error.
```

Comment:
Issue instead of Jira?

Reply:
Right you are.

Reply:
On a second thought - this whole note is outdated. Since there's no more internal registry, and commits should be validated prior to their submission (merging), there shouldn't (save for outlandish reasons) be any more problems pulling the images.